src/perf/lib/RpsClient.h

```diff
@@ -22,14 +22,16 @@ struct RpsWorkerContext;
 class RpsClient;
 
 struct StreamContext {
+    HQUIC Handle {nullptr};
     StreamContext(
         _In_ RpsConnectionContext* Connection,
         _In_ uint64_t StartTime)
         : Connection{Connection}, StartTime{StartTime} { }
+    ~StreamContext() noexcept { if (Handle) { MsQuic->StreamClose(Handle); } }
```

Comment:
I can revert this too if you think it is not useful now @nibanks

Reply:
Yeah, let's just remove everything not absolutely necessary